specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/streams/application/fetch/filter.extracted.header/client.rpt

```diff
@@ -75,7 +75,7 @@ read zilla:data.ext ${kafka:matchDataEx()
                              .fetch()
                                  .partition(0, 2, 8)
                                  .header("header1", "value1")
-                                 .header("correlation-id", "12345")
+                                 .header("correlation-id", "1234")
```

Comment:
Just curious, what prompted this change?

Reply:
The script we are using has 4 bytes `key1` and the TestConverter was returning `12345` which has 5 bytes and it was causing testware issue. I just changed TestConverter to return 1234 to match number of bytes in server script.